ui/src/components/table/TableWrapper.tsx

```diff
@@ -61,13 +95,16 @@ function TableWrapper({
                 obj[service.name] = tmpObj;
             }
         });
+
         setRowData(obj);
         setLoading(false);
     };
 
     const fetchInputs = () => {
-        const requests = [];
-        services.forEach((service) => {
+        const requests: Promise<{
+            data: { entry: { name: string; content: Record<string, string>; id: string }[] };
+        }>[] = [];
+        services?.forEach((service) => {
```

Comment:
changed, generally i did not do many updates here just moved it to ts as that whole file needs some refactoring as there are weird behaviours. But maybe as you suggested it is better to at least do a small improvement instead of one big change.